lib/flags.ml

```diff
@@ -189,6 +189,11 @@ let print_hyps_limit = ref (None : int option)
 let set_print_hyps_limit n = print_hyps_limit := n
 let print_hyps_limit () = !print_hyps_limit
 
+(* Flag for compact display of goals *)
+
+let compact_context = ref false
+let is_compact_context() = !compact_context
```

Comment:
@Matafou: This is the usual dilemma between centralized information and easy extensibility. At least, you can list the existing options with "Print Options." (then grep in ml files for the Coq name of the option to know the ml name of the associated flag).